man/tpm2_encryptdecrypt.1.md

```diff
@@ -74,11 +70,20 @@ specified symmetric key.
 
 # EXAMPLES
 
+# Create an AES key
+```bash
+tpm2_createprimary -c primary.ctx
+tpm2_create -C primary.ctx -Gaes128 -u key.pub -r key.priv
+tpm2_load -C primary.ctx -u key.pub -r key.priv -c key.ctx
 ```
-tpm2_encryptdecrypt -c 0x81010001 -p abc123 -i <filePath> -o <filePath>
-tpm2_encryptdecrypt -c key.dat -p abc123 -i <filePath> -o <filePath>
-tpm2_encryptdecrypt -c 0x81010001 -p 123abca  -i <filePath> -o <filePath>
-tpm2_encryptdecrypt -c 0x81010001 --enable-pkcs7-padding -i <filePath> -o <filePath>
+
+# Encrypt and Decrypt some data
+```bash
+echo "my secret" > secret.dat
+tpm2_encryptdecrypt -c key.ctx -o secret.enc secret.dat
+tpm2_encryptdecrypt -d -c key.ctx -o secret.dec secret.enc
+cat secret.dec
```

Comment:
Do we want to compare the results or simply print the output to stdout in the example section.